test/features/pagecontent/save_api.js

```diff
@@ -251,10 +272,28 @@ describe('page save api', function() {
         });
     });
 
+    it('should forbid external usage', function() {
```

Comment:
The fact that this test passes means that there's something wrong with the code :) Hint: it's actually related to my comment about adding `|| req.socket.remoteAddress`